pkg/build/registry/buildlog/rest.go

```diff
@@ -125,10 +125,10 @@ func (r *REST) waitForBuild(ctx kapi.Context, build *api.Build) error {
 			}
 			switch obj.Status {
 			case api.BuildStatusCancelled:
-				errchan <- fmt.Errorf("Build %s/%s was cancelled", build.Namespace, build.Name)
+				errchan <- fmt.Errorf("Build %s/%s was cancelled. %s", build.Namespace, build.Name, buildutil.NoBuildLogsMessage)
```

Comment:
@smarterclayton even if its the name of the resource ?
eg: https://github.com/openshift/origin/blob/master/pkg/build/generator/generator.go#L128